aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -35,19 +36,21 @@ Set up a CI build for an ASP.NET Core app, then create a continuous deployment r
 [Azure Web App sandbox](https://github.com/projectkudu/kudu/wiki/Azure-Web-App-sandbox)  
 Discover Azure App Service runtime execution limitations enforced by the Azure Apps platform.
 
-::: moniker range=">= aspnetcore-2.0"
-
 ## Application configuration
 
-The following NuGet packages provide automatic logging features for apps deployed to Azure App Service:
+### Platform
+
+Runtimes for 64-bit (x64) and 32-bit (x86) apps are present on the Azure App Service and supported for ASP.NET Core apps. The [.NET Core SDK](/dotnet/core/sdk) available on App Service is 32-bit, but you can deploy 64-bit apps using the [Kudu](https://github.com/projectkudu/kudu/wiki) console or via [MSDeploy with a Visual Studio Azure App Service publish profile or CLI command](xref:host-and-deploy/visual-studio-publish-profiles).
```

Comment:
This is confusing. Unless you have native dependencies (which most projects don't have), you rarely deploy a specific bitness of your application. You always deploy a portable app